libs/remix-tests/tests/examples_0/assert_greaterThan_test.sol

```diff
@@ -29,8 +29,4 @@ contract AssertGreaterThanTest {
     function greaterThanIntUintPassTest() public {
         Assert.greaterThan(int(10), uint(2), "greaterThanIntUintPassTest passes");
     }
-
-    function greaterThanIntUintFailTest() public {
```

Comment:
this doesn't compile?

Reply:
In the latest version, `uint` doesn't accept negative values